mlflow/cli.py

```diff
@@ -165,7 +165,9 @@ def _validate_server_args(gunicorn_opts=None, workers=None, waitress_opts=None):
                    "Default: " + DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH)
 @click.option("--port", "-p", default=5000,
               help="The port to listen on (default: 5000).")
-def ui(backend_store_uri, default_artifact_root, port):
+@click.option("--host", "-h", default="127.0.0.1",
```

Comment:
Rename `host` to `host-ip-address` to make it clear. Also fix the help to reflect this change.

Reply:
I let the parameter named `host` to be consistent with the `server` command. I also updaded the decorators to the generic `utils.cli_args` host and port ones.

Reply:
ah yes! And thanks for using `@cli_args` decorators